plugins/lando-engine/lib/compose.js

```diff
@@ -227,15 +227,15 @@ exports.build = function(compose, project, opts) {
 /*
  * Run docker compose pull
  */
-exports.pull = function(compose, project, opts) {
+exports.pull = (compose, project, opts) => {
 
   // Let's get a list of all our services that need to be pulled
   // eg not built from a local dockerfile
-  var images = _.filter(_.keys(_.get(opts, 'app.services'), []), function(service) {
-    return !_.has(opts.app.services, service + '.build');
```

Comment:
I assume this was an automatic correction, though I am not sure if implicit returns work here.